janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/AbstractCassandraStoreManager.java

```diff
@@ -337,4 +315,20 @@ protected String determineKeyspaceName(Configuration config) {
         if ((!config.has(CASSANDRA_KEYSPACE) && (config.has(GRAPH_NAME)))) return config.get(GRAPH_NAME);
         return config.get(CASSANDRA_KEYSPACE);
     }
+
+    private Map<String, String> convertOptions(ConfigOption<String[]> optionsKey, Configuration config){
```

Comment:
This method converts the options from a `String[]` into a `Map<String, String>`, so how about conveying that with its name? Something like `convertOptionsToMap` maybe? That way, it won't be necessary any more to read the code of this method as the name already tells everything it does. Right now, I would need to read the code as I wouldn't immediately see that the method converts the options into a map.

Reply:
@FlorianHockmann thank you for your reviews! I've updated a method name.